AllReadyApp/Web-App/AllReady/Services/SendGridDevelopmentWeb.cs

```diff
@@ -21,44 +23,9 @@ public Task DeliverAsync(ISendGrid message)
             var client = new SmtpClient();
             var resultObject = new object();
             client.DeliveryMethod = SmtpDeliveryMethod.SpecifiedPickupDirectory;
-            client.PickupDirectoryLocation = _config["DevelopmentEmailFolder"];
+            client.PickupDirectoryLocation = _settings.EmailFolder;
             client.SendAsync(new MailMessage(message.From.Address, message.To.FirstOrDefault().Address, message.Subject, message.Html), resultObject);
             return Task.FromResult(resultObject);
         }
     }
-
-    public static class ITransportAdapter
```

Comment:
This wasn't used anywhere - I had been under the assumption that @MisterJames had made changes that make this obsolete?